### PR TITLE
Fixed link color to meet 4.5:1 contrast

### PR DESCRIPTION
**Pull request by betrue-final-final**
#### Pull request checklist

- [x ] Addresses an existing issue: #1009

#### Description of changes

Changed link color for messagebar to themeDark so it meets 4.5:1 contrast

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
   :global(.ms-Link) {
     font-size: $ms-font-size-s;
+    color: $ms-color-themeDark;
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss]: Is this different from the text color?

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss]: yes, it's a dark blue

Comment (betrue-final-final) [inline packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss]: It's darker than standard link color. It's according to the spec

**Comment by cschleiden** [comment]
Do we want a changefile for this?

**Pull request merged**